client/profile-wizard/style.scss

```diff
@@ -424,6 +454,18 @@
 	}
 }
 
+.woocommerce-profile-wizard__submit {
+	margin-bottom: $gap-small;
+}
+
+.woocommerce-profile-wizard__tooltip-icon {
+	color: $studio-gray-60;
+	display: flex;
+	align-items: center;
+	margin-left: 8px;
+	cursor: pointer;
```

Comment:
I see the pointer on the designs, but it feels wrong to me. Here is Github's:

![Screen Shot 2020-07-09 at 1 22 36 PM](https://user-images.githubusercontent.com/1922453/86986495-f0378d00-c1e7-11ea-9ff6-509e139db843.png)

Maybe `cursor: help;` ?

Reply:
I agree the cursor feels wrong. Will fix.